sufia-models/app/jobs/copy_visibility_job.rb

```diff
@@ -0,0 +1,21 @@
+class CopyVisibilityJob 
+  def queue_name
+    :permissions
+  end
+
+  attr_accessor :id
+
+  def initialize(id)
+    self.id = id
+  end
+
+  def run
+    work = ActiveFedora::Base.load_instance_from_solr(id)
+    if work.respond_to?(:generic_files)
+      work.generic_files.each do |file|
+        file.visibility = work.visibility
```

Comment:
I agree.  Unless @mjgiarlo is opposed, let me review the way sufia implements edit access and I with integrate that into this story too.